plugins/metrics/runtime/internal/procstats/procstats.go

```diff
@@ -0,0 +1,243 @@
+package procstats
```

Comment:
It doesn't look like this internal package is used anywhere in here.

Reply:
@zlozano, nah it isn’t used. Just example code (From segment.io stats) I copied for reference. Will remove. Didn’t notice as unused vendor code isn’t copied in by go mod